src/ipc/ipc4/dai.c

```diff
@@ -191,6 +191,8 @@ void dai_dma_release(struct comp_dev *dev)
 			mailbox_sw_regs_write(llp_reg_offset, &slot, sizeof(slot));
 		}
 
+		dma_stop(dd->chan);
```

Comment:
I would put a comment here explaining why the DMA may still be running at this point.

Reply:
sure